vonk/configuration/appsettings.rst

```diff
@@ -342,6 +352,8 @@ But you can disable interactions by removing them from these lists.
         "WholeSystemInteractions": "capabilities, batch, transaction, history, search, $validate"
     },
 
+If you implement a custom operation in a plugin, you should also add the name of that operation at the correct level. E.g. add ``$convert`` to ``TypeLevelInteractions`` to allow ``<base>/<resourcetype>/$convert``.
```

Comment:
Earlier in this file: missing a description for the log levels

Reply:
How about "Refer to :ref:`configure_log` for information on setting log levels." on line 97?